numpy/core/_add_newdocs.py

```diff
@@ -8009,22 +8022,65 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'float128',
     """
-    128-bit floating-point number. Character code: 'g'. C long float
+    128-bit floating-point number. Character code: ``'g'``. C long float
     compatible.
 
     """)
 
 add_newdoc('numpy.core.numerictypes', 'int8',
-    """8-bit integer. Character code ``b``. C char compatible.""")
+    """
+    8-bit integer. Character code ``'b'``. C char compatible.
+
+    """)
 
 add_newdoc('numpy.core.numerictypes', 'int16',
-    """16-bit integer. Character code ``h``. C short compatible.""")
+    """
+    16-bit integer. Character code ``'h'``. C short compatible.
+
+    """)
 
 add_newdoc('numpy.core.numerictypes', 'int32',
-    """32-bit integer. Character code 'i'. C int compatible.""")
+    """
+    32-bit integer. Character code ``'i'``. C int compatible.
```

Comment:
Not true - on my machine, the character code is `l`.

If we want to add these docs, I think we should do it to `byte`/`short`/`intc`/`int_`/`longlong`, not the sized aliases